fighters/ryu/src/status.rs

```diff
@@ -1,6 +1,25 @@
 use super::*;
 use globals::*;
 
+pub trait Vector3fExt {
+    fn mag(&self) -> f32;
+    fn normalize(&self) -> Self;
+  }
+  
+  impl Vector3fExt for Vector3f {
+    fn mag(&self) -> f32 {
+      (self.x.powi(2) + self.y.powi(2) + self.z.powi(2)).sqrt()
+    }
+    fn normalize(&self) -> Self {
+      let mag = self.mag();
+      Self {
```

Comment:
no.

move this to our regular extensions on Vector3f please, if they aren't already there. Don't just drop this in a status